File: 3194-find-words-containing-character/find-words-containing-character.java
class Solution {
    public List<Integer> findWordsContaining(String[] words, char x) {
        List<Integer> list =new ArrayList<>();

        for(int i=0;i<words.length;i++){
            String curr=words[i];
            for(int j=0;j<curr.length();j++){
                if(curr.charAt(j)==x){
                    list.add(i);
                    break;
                }
            }
        }
        return list;
    }
}

File: 3194-find-words-containing-character/find-words-containing-character.py
class Solution(object):
    def findWordsContaining(self, words, x):
        """
        :type words: List[str]
        :type x: str
        :rtype: List[int]
        """
        list=[]
        for i in range(len(words)):
            if(x in words[i]):
                list.append(i)

        return list
        